Yoast/wordpress-seo
# Wrap Snippet Preview in its own collapsible container

## Summary

This PR can be summarized in the following changelog entry:

* Adds a collapsible container to the metabox to support the Snippet Preview / Snippet Editor.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Ensure that you have Yoast components installed (`yarn upgrade yoast-components`).
* Create a new post or edit an existing one in Gutenberg.
* Scroll down to the metabox and notice that there's a collapsible present for the snippet preview / snippet editor.
* Open the collapsible and determine that the preview and editor still work as expected.

## Quality assurance


Fixes #10455
